CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## [UNRELEASED] 1.8.7
+## [UNRELEASED] 1.9.0
```

Comment:
Yes, in `1.8.6` it was also forgotten, but let's put it here as it is the convention throughout the entire changelog 
```suggestion
## [UNRELEASED] neptune 1.9.0
```